www/scripts/productlist/ListOfProducts.js

```diff
@@ -167,10 +167,11 @@ function (declare, domClass, domConstruct, ItemFileWriteStore, topic,
 
     onRowClick : function (evt) {
       var item = this.getItem(evt.rowIndex);
+      var databaseStatus = item.databaseStatus === 0;
```

Comment:
What does this line, and the one in L174 do? This initialises the variable to a boolean checking if the API-reported status is `0`?

Reply:
I think this was left there this way by accident.